### PR TITLE
[NCL-4810] Add support for custom parameters for Gradle builds - 1.7

### PR DESCRIPTION
**Pull request by goldmann**
This reuses the `CUSTOM_PME_PARAMETERS` variable and is subject
for refactoring after 1.7 release.

### CODE REVIEW

File: repour/adjust/gradle_provider.py [hunk 1]
```diff
@@ -39,6 +39,10 @@ def adjust(work_dir, extra_adjust_parameters, adjust_result):
             temp_build_parameters.append(
                 "-DrestRepositoryGroup=" + specific_indy_group)
 
+        extra_parameters, subfolder = yield from util.get_extra_parameters(extra_adjust_parameters)
+
+        repo_dir = os.path.join(repo_dir, subfolder)
```
Comment (vibe13) [inline repour/adjust/gradle_provider.py]: sorry I am 0 on python, I am missing why LN#44 is needed? repo_dir seems not used?

Comment (goldmann) [reply repour/adjust/gradle_provider.py]: Well, this is a good catch. It's a copy/paste error, in PME it is `repo_dir`, in GME it's `work_dir`. Fixing a second.

Comment (goldmann) [reply repour/adjust/gradle_provider.py]: This should be fixed now.

**Comment by vibe13** [comment]
@goldmann looks good to me, maybe @thescouser89 wants to take a look as well? Thanks!

**Pull request merged**